CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if (CMAKE_BINARY_DIR STREQUAL CMAKE_SOURCE_DIR)
     message(FATAL_ERROR "Building in-source is not supported! Create a build dir and remove ${CMAKE_SOURCE_DIR}/CMakeCache.txt")
 endif()
 
+project(openrct2)
```

Comment:
The sub includes are not modules, they represent the different projects within the workspace. So each include is a cmakelist which sets `project()` to the output its creating.